modules/cloud-function-v2/outputs.tf

```diff
@@ -21,11 +21,6 @@ output "bucket" {
   )
 }
 
-output "bucket_name" {
```

Comment:
Removing the output prevents consumers of the module create implicit dependency on the bucket created. I would keep that.